PlayerTwo/playerGuessInput.js

```diff
@@ -31,7 +31,7 @@ const isCorrectGuess = () => {
 
     } else {
         document.getElementById("input-result").innerHTML = "incorrect";
-        lives -= 1;
+        // lives -= 1; this is not defined
```

Comment:
We get error in the console because `lives` one is not defined anywhere

Reply:
We probably do not need lives anymore because we have a timer instead, maybe I should just take them out completely?